src/main/java/org/folio/dew/batch/JsonFileWriter.java

```diff
@@ -43,9 +43,7 @@ public String doWrite(Chunk<? extends U> items) {
       } else {
         lines.append(marshaller.marshal(item.getOriginal()));
       }
-      if(iterator.hasNext()) {
-        lines.append('\n');
-      }
+      lines.append('\n');
```

Comment:
Can be extracted to constant - separator/delimiter.

Reply:
Extracted.